python/py-url-normalize/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-url-normalize
+version             1.4.3
+revision            0
+categories-append   devel net
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     27 36 37 38 39 310
+
+maintainers         nomaintainer
+
+description         URI Normalization function
+
+long_description    ${description}
+
+homepage            https://github.com/niksite/url-normalize
+
+checksums           rmd160  2b189448fd045c9a9a47cb3d2af04e6f3dfea1c0 \
+                    sha256  d23d3a070ac52a67b83a1c59a0e68f8608d1cd538783b401bc9de2c0fac999b2 \
+                    size    6024
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        port:py${python.version}-setuptools
+
+    livecheck.type      none
+}
```

Comment:
the lines in this `if`-block can be aligned in the same was as the rest of the Portfile

Reply:
I think I got this one - but will click resolve only after you confirm